tutorials/schedule-dataflow-jobs-with-cloud-scheduler/index.md

```diff
@@ -67,7 +70,54 @@ EOT
 }
 ```
 
-Afterwards you are all set up! 
+## Instructions
+
+The following step-by-step instructions can be used to create a sample Dataflow pipeline with Cloudbuild.
+
+First, open Cloud Shell and clone the repository.
+
+```
+git clone https://github.com/GoogleCloudPlatform/community
+cd community/tutorials/schedule-dataflow-jobs-with-cloud-scheduler/scheduler-dataflow-demo/
+```
+
+Create a bucket on Google Cloud Storage, which will be used to store terraform states and Dataflow templates.
+Replace BUCKET_NAME and PROJECT_ID with your own choice.
```

Comment:
You can use ${GOOGLE_CLOUD_PROJECT} for project ID, this is predefined in Cloud Shell.

Also suggest you define $REGION here, so it can be used later.

Reply:
cool. going to define the region in the below section. I need to make a note about region, because it depends on where the app engine locates. As a result, it is not something users can really choose.